content/docs/redhat/acm/submariner/rosa/rosa-submariner.md

```diff
@@ -0,0 +1,706 @@
+# Deploy ACM Submariner for connect overlay networks of ROSA clusters
+
+Author: [Roberto Carratalá](https://github.com/rcarrata)
+
+Submariner is an open source tool that can be used with Red Hat Advanced Cluster Management for Kubernetes to provide direct networking between pods and compatible multicluster service discovery across two or more Kubernetes clusters in your environment, either on-premises or in the cloud.
+
+This article describes how to deploy ACM Submariner for connecting ROSA clusters overlay networks.
+
+NOTE: ACM Submariner for ROSA clusters only works from ACM2.7 onwards!
```

Comment:
Does it only work with ACM 2.7 or newer or is it only available with ACM 2.7 or newer? The former implies it's available and broken.

In either case, you should word the ACM requirement as "with ACM 2.7 or newer".

Reply:
Only works with ACM 2.7 or newer. I will rephase the sentence to make it more clear.